lib/sizzle.patched.js

```diff
@@ -1572,6 +1575,8 @@ Expr = Sizzle.selectors = {
 	}
 };
 
+Expr.pseudos["has-text"] = Expr.pseudos["contains"];
```

Comment:
Why can't we do it on our side? Patching sizzle to add another pseudo-class seems redundant.